frontend/src/lang/es.ts

```diff
@@ -151,6 +151,7 @@ export const es: LanguageConfig = {
           addNumber: "Agregar otro número",
           addEmail: "Agregue otra dirección de correo electrónico",
           email: "Dirección de correo electrónico",
+          additionalEmail: "Dirección de correo electrónico",
```

Comment:
Do we have a more specific string for additionalEmail? It's the same for both email and additionalEmail on es.ts, is that intentional?